### ISSUE TITLE
Tracks: wcadmin_settings_change not working on the Payments tab

### ISSUE THREAD

**Issue by pmcpinto** [woocommerce/woocommerce-admin#5875]
**Describe the bug**
The `wcadmin_settings_change` is not being triggered when you make an update on the Payments tab (Settings)

**To Reproduce**

1. Go to Settings > Payments (make sure that you have event tracking enabled)
2. Toggle one or more payments options
3. Save changes
4. See that `wcadmin_settings_change` wasn't triggered in the network tab. 

**Expected behavior**
Trigger `wcadmin_settings_change` when the user saves any change in the Payments tab. The `tab` prop should have the `checkout` value

**Comment by pmcpinto** [comment]
@timmyc I suppose that this needs to be implemented in WC Core, so I'm wondering if we still have time to include a fix for 4.9. Assuming that the fix is straightforward. If not let's target 5.0

**Comment by timmyc** [comment]
Confirmed this is not working on a ecom plan site. I tried toggling COD payment type, and also adjusting values of the COD payment type description and did not see the tracks event record at all. Worth mentioning this is a server side track, so nothing will be shown in the network tab. Rather I was using the tracks viewer for my wpcom user account to test.

Associated logic for this particular track [lives here](https://github.com/woocommerce/woocommerce/blob/085b4b53fae187b0669714bc9da4e3084ba318d8/includes/tracks/events/class-wc-settings-tracking.php#L102) so we should probably move the issue to the woo core repo.
